db/migrate/20180928235853_create_disability_compensation_job_statuses.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDisabilityCompensationJobStatuses < ActiveRecord::Migration
+  def change
+    create_table :disability_compensation_job_statuses do |t|
+      t.integer :disability_compensation_submission_id, null: false
+      t.string :job_id, unique: true, null: false
+      t.string :job_class, null: false
+      t.string :status, null: false
+      t.string :error_message
+      t.column :updated_at, :datetime, null: false
```

Comment:
Is there a reason you didnt include a `created_at` row, or is it just because it doesnt really matter for a registry?

Reply:
yes and yes, the reason was that Postgres SQL call that upsert makes didn't play nicely with created_at. I'll take another shot at debugging it now that everything else is in.

Reply:
Actually I remember now, upsert needs the created_at field in the SQL to enter it, but we're not sure which is the first upsert. Later when the record is updated again the performance gained from not first finding the data means you don't know created_at (just to write it back). So yes doesn't matter because it's a registry 😄and it's faster without.